src/commands/account/create_account/fund_myself_create_account/sign_as/mod.rs

```diff
@@ -0,0 +1,446 @@
+use dialoguer::{console::Term, theme::ColorfulTheme, Input, Select};
+use serde_json::json;
+use std::str::FromStr;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = crate::commands::account::create_account::CreateAccountContext)]
+#[interactive_clap(skip_default_from_cli)]
```

Comment:
Did you implement `from_cli` from scratch only to pass the context to `crate::network_for_transaction::NetworkForTransactionArgs::from_cli()`? You should be able to implement `input_signer_account_id` and `impl From<CreateAccountContext> for GlobalContext` for

```suggestion
#[interactive_clap(input_context = crate::commands::account::create_account::CreateAccountContext, output_context = crate::GlobalContext)]
```

Reply:
done